bower.json

```diff
@@ -2,7 +2,7 @@
     "name": "adapt-hotgrid",
     "repository": "git://github.com/cgkineo/adapt-hotgrid",
     "framework": "^2.0.0",
-    "version": "2.0.0",
+    "version": "3.0.0",
```

Comment:
I've made this 3.0.0 as this PR could potentially introduce breaking changes with the `<a>` to `<button>` tag conversion.